tests/framework-tests/core/html/Caching-test.html

```diff
@@ -21,6 +21,10 @@
         <script type="text/javascript" src="../../../lib/mockjax/js/jquery.mockjax.js"></script>
         <script type="text/javascript" src="../../../test-core/jqUnit/js/jqUnit.js"></script>
 
+        <script src="/testem.js"></script>
```

Comment:
Would be helpful to indicate these .js files are for reporting coverage data, just like what you did with some other test files such as `UploaderCompatibility-test.html`, the one above. This comment applies across to other files that have the same issue.